k-distribution/pl-tutorial/2_languages/1_simple/1_untyped/tests/threads/threads_04.simple.out

```diff
@@ -1,3 +1,23 @@
-  S ==K "2 3 "
+  {
+    S:String
+  #Equals
+    "2 3 "
+  }
 #Or
-  S ==K "3 2 "
+  {
+    S:String
+  #Equals
+    "3 2 "
+  }
+#Or
```

Comment:
I expected us to lose some behaviors in some cases, but why have we gained behaviors?

Reply:
I'm not sure why the original output had only two behaviours. Looking at both the semantics of the language, and the comment in the program, it seems that 4 behaviours are to be expected.